py_mmd_tools/cf_elements.yaml

```diff
@@ -13,7 +13,7 @@ history:
 featureType:
```

Comment:
Her skal det vel være feature_type?

Reply:
Ser ut som "featureType" fra https://cfconventions.org/Data/cf-conventions/cf-conventions-1.10/cf-conventions.html#_features_and_feature_types - hva er rett @magnarem ?

Reply:
I think the bokeh application only supports datasets which conforms to the timeSeries or timeSeriesProfile feature types as of now. @epifanio knows maybe more.

Reply:
Ja beklager. i netcdf skal global attribute være featureType:

i solr heter feltet feature_type, så jeg blanda. beklager.